### ISSUE TITLE
Kia Forte can error due to missing SCC messages

### ISSUE THREAD

**Issue by AskAlice** [commaai/openpilot#1670]
**Describe the bug**
So I've personally experienced a pair of permanent can error alerts as well as device sensor alerts with 0.8. Presumably this began around 0.7.6. On 0.7.5 it was working great up until a couple days before release, then I started getting 'can error check connections' for that.

None of these seem to provide me with any useful tracing in tmux.

I do not have long control, so generally my build requires a couple tweaks before it can engage, but @KyleGreenlaw reported to me that with his vehicle that does have SCC, he was unable to get the latest and greatest running on HKG as well.

![image](https://user-images.githubusercontent.com/1855905/84196833-0d006980-aa5e-11ea-8a12-bf446474e92b.png)
![image](https://user-images.githubusercontent.com/1855905/84196869-1db0df80-aa5e-11ea-8ce5-5371567fd3d8.png)

eb1f2dd16c2313e6|2020-06-09--02-56-34
https://my.comma.ai/eb1f2dd16c2313e6/1591692995041/1591693110984

i spent half a day or so yesterday after work working on sending SCC messages, and the messages were sending, the menu popped up and all, I just couldn't engage because of the same issue that occurs with a clean diff.

**How to reproduce or log data**

https://my.comma.ai/cabana/?route=eb1f2dd16c2313e6%7C2020-06-08--21-01-53&exp=1623208962&sig=EVY5xqp7AnVa%2B%2BhulhQ7f8W9Ay6nlqDUcpbOfwMmQ5c%3D&max=1&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Feb1f2dd16c2313e6%2F389e75adb6f910a566778fe41c2ddaa2_2020-06-08--21-01-53

get in a 2019 forte or a 2018 elantra and run anything newer than 0.7.4

**Expected behavior**
`alerts = []`
**Device/Version information (please complete the following information):**
 - Device: [e.g. EON/EON Gold] comma two, and eon (both tested)
 - Dongle ID: [e.g. 77611a1fac303767, can be found in Settings -> Device -> Dongle ID] eb1f2dd16c2313e6
 - Version: [e.g. 0.6.4] 1a2411be0fbbc0db91914ff7dfc29e156471d7ab
 - Car make/model [e.g. Toyota Prius 2016] 2019 kia forte

**Additional context**
It appears to me that I haven't been able to engage since willem added the firmware id queries, but there could have just been other HKG dev going on around that same time as that was a goal of 0.7.5. That being said, the alerts are different on 0.7.5 compared to 0.7.6+

Another potential culprit could be the DBC file recently changed. Perhaps TK's stuff somehow broke existing code? It wouldn't make any sense to me though as it generally spams in tmux when the struct is different, and i made sure all the signal names were correct based on the diff

my eyes point towards the panda code

**Comment by pd0wm** [comment]
It's because your car doesn't have SCC. Currently openpilot does not perform longitudinal control and expects the SCC messages for part of the carState. Specifically the `SCC11` and `SCC12` message. When I merged the hyundai port into openpilot I added checks that requires all the messages that are used in carState to be present (https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/carstate.py#L199). The hkg community fork works for you because those checks are not present and uses some default values for those fields.

So it seems like until we add KHG longitudinal support, your car is not supported by openpilot.
